communities/aston-css/community-assessment.md

```diff
@@ -0,0 +1,159 @@
+
+- Is there parking, and if so, is it free?
+
+Yes, and no. But this is not really relevant for most members.
+
+- Is there public transportation available to and from meet-ups?
+
+People are usually already in the University building so not applicable. We are a 10 minute (Scott pace) walk from the central train stations in Birmingham for those not on campus. Very accessible.
+
+- Is the location accessible to those both on and off campus?
+
+Yes, see above.
+
+- How much does it cost to use the space?
+
+Totally free!
+
+- How flexible is the space for different types of presentations and meetings?
+
+It has a whole load of chairs and tables, but they're all on legs and can be moved about very easily (and are regularly). We can expand the space if needed, and occupy the student software company office (there's a big double door that opens up to join the two). The room has a projector and speakers.
+
+- How many people can it fit?
+
+I've never tried to cram it full before, but maybe 30 people before it gets too much. And that's a bit of a squeeze. As a result, we quite often book out other rooms in the University for larger events. This works pretty well, especially now that we've got a contact in the department who's happy to book rooms for us any time, in an instant!
+
+- Is there comfortable seating for all shapes and sizes?
+
+Yes. We can also steal sofas from the lobby outside (haven't thought about trying this yet, will report back on whether we get told off or not).
+
+- Is there water or snacks available nearby?
+
+All kinds, yes! We usually buy some for organised events, but for the rest of the time, there are cups for water, a kettle, and a cafeteria/vending machines for food.
+
+### Communication
+
+- How does your community get in touch with one another?
+
+At the moment, by word of mouth and in our Facebook group. The committee uses a Slack team.
+
+- Is there an email group, a GitHub organization, a Twitter account, a Facebook group, a chat room or channel?
+
+We have a mailing list through the student union (but it's a pain to use). There's also a GitHub organisation, but it's not really used. We have various social accounts for posting news/advertising events (Facebook, Twitter).
+
+- Could your organization benefit from one or more of those, or another way of communicating?
+
+I've just stepped in as head of the society, and one of the first things I'm hoping to do is open up our Slack team to all members. At the moment it's just the committee, and it needs a bit of a tidy up before making it open!
+
+### Culture
+
+- How welcoming is your community to new people and new ideas?
+
+There are certainly people who try to be welcoming (I consider myself one), but the society as a whole is ripe with inside jokes and memes. I really don't like this culture, and we're going to try and get rid of over the next year (not 100% sure on a plan to go about this yet though).
+
+- What happens if there is a disagreement?
+
```

Comment:
We have kind of a stub of one pinned to our Facebook group, but that's more to do with what kind of content to post. Having a proper CoC is something we've considered a bit in the past, but never really put much effort towards. Maybe it's time!

Our website has been offline for like 2 years now, and we're in the process of putting a new one together, so now seems like a good time to be introducing new conventions and the like.